pkg/storageops/vsphere/vsphere.go

```diff
@@ -409,3 +460,111 @@ func (ops *vsphereOps) renewVM(ctx context.Context, vm *vclib.VirtualMachine) (*
 	vmObj := vm.RenewVM(client)
 	return &vmObj, nil
 }
+
+// recommendedDatastore recommends a datastore to use for the given storage pod by
+// quering the storage resource manager
+func recommendDatastore(
```

Comment:
nit: add a reference from where this function was taken.

Reply:
done